core/validators/signatures.go

```diff
@@ -106,6 +108,49 @@ type NodeIDAddress struct {
 	SubmitterAddress string
 }
 
+func (s *ERC20Signatures) OnTick(ctx context.Context, _ time.Time) {
+	s.notary.OfferSignatures(types.NodeSignatureKindERC20MultiSigSignerAdded, s.offerValidatorAddedSignatures)
+	s.notary.OfferSignatures(types.NodeSignatureKindERC20MultiSigSignerRemoved, s.offerValidatorRemovedSignatures)
+}
+
+func (s *ERC20Signatures) offerValidatorAddedSignatures(id string) []byte {
+	if s.isValidatorSetup {
+		return nil
+	}
+
+	sd, ok := s.pendingSignatures[id]
```

Comment:
I don't think this is right, the keys for `s.pendingSignatures` are ethereum addresses. It's the `issuedSignatures` map that has the notary resourceID as its keys, but then it maps to nothing which is unhelpful.

So I think we might need to do something where `s.issuedSignatures -> {eth-address, submitter-address}` and then you can get do `s.pendingSignatures[s.issuedSignatures[id].EthAddress]` to get the other bits, or something like that.